jxl/src/headers/extra_channels.rs

```diff
@@ -54,7 +54,7 @@ pub struct ExtraChannelInfo {
     // TODO(veluca93): if using Option<bool>, this is None when all_default.
     #[condition(ec_type == ExtraChannel::Alpha)]
     #[default(false)]
-    alpha_associated: bool,
```

Comment:
did you accidentally delete this line or is github just confused?

Reply:
I made it `pub`, which GitHub presents as one `-` and one `+`.